R/cf.R

```diff
@@ -355,6 +417,112 @@ gen.block.array <- function(n, R, l, endcorr=TRUE) {
   return(list(starts = st, lengths = lens))
 }
 
+#' Computes the samples for reweighted correlation function
+#'
+#' @param cf `cf` object.
+#' @param rw `rw` object.
+#' @param nsamples Integer (Number of stochastic samples)
+#' @param boot.R Integer
+#' @param boot.l Integer
+#' @param seed Integer
+#' @param sim string
+#' @param endcorr boolean
+#' @export
+bootstrap_rw.cf <- function(cf, rw, nsamples=1, boot.R=400, boot.l=2, seed=1234, sim="geom", endcorr=TRUE) {
```

Comment:
Not sure about the necessity of ```nsamples``` here. Shouldn't the reweighting factors already be averaged at this stage?

Reply:
yes, I correct this